### PR TITLE
Refactor header with tables at 600px width and make responsive

### PR DESCRIPTION
**Pull request by eliasfang**
### Administrative Info
Monday Board ID: 940408300
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Refactored all of my code so that everything consisted of HTML tables
- Moved all CSS into newsletter.css (I commented out existing CSS for now)
- Added a media query for mobile widths less than 600px
- Resized fonts/images through trial and error by using Chrome inspect feature
- Changed size of everything to fit the maximum width of 600px (this can be changed if we don't like it)
- Added hyperlinks to the logo and the social media

### Testing
How did you confirm your changes worked? 
- I sent test emails to myself and checked how it looked on Desktop Gmail, Mobile Gmail, and Mobile Apple Mail
- I found that on mobile, Gmail was not responsive but Apple Mail was. This is probably an issue for another task.
- Clicking on the logos and images links to the socials on a separate page

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| Desktop Gmail | Mobile Gmail (Issue) | Mobile Apple Mail | Mobile Apple Mail Dark |
| --- | --- | --- | --- |
| image | Image from iOS (1) | Image from iOS | File |

### CODE REVIEW

**Review by wes-chen — COMMENTED** [review]
So @eliasfang will the newsletter have a white background outside of the email? How does this look in dark mode? What does it look like if we squish the email to less than 600px?

**Comment by eliasfang** [comment]
> So @eliasfang will the newsletter have a white background outside of the email? How does this look in dark mode? What does it look like if we squish the email to less than 600px?

- Yes, the newsletter will have a white background outside of the email for light mode and a dark gray background for dark mode. In dark mode, it still has the inverted colors (I added the screenshot above). The phone screenshots are all less than 600px if that's what you're asking. But if the user were to resize the window, it would retain responsiveness.

File: newsletter.css [hunk 1]
```diff
@@ -266,4 +262,159 @@ footer {
     margin-top: 6.393125em;
     margin-right: 2.5em;
     margin-bottom: 10.0625em;
+} */
```
Comment (wes-chen) [inline newsletter.css:265]: can we remove these comments? if we commit this in then @shravan1k 's code will break. alternatively you can prefix those css tags with `sk-` like you did for yours and let @shravan1k know

**Review by wes-chen — CHANGES_REQUESTED** [review]

File: newsletter.css (hunk 1 above)
Comment (eliasfang) [reply newsletter.css:265]: Sounds good. I made sure Shravan knows!

**Review by wes-chen — APPROVED** [review]
lgtm, i think @shravan1k 's files will now break without an update to his html files. @eliasfang i have opened another monday task for the mail responsiveness task.

**Review by shravan1k — APPROVED** [review]
LGTM. I will make the updates to my html files in my next commit.

**Pull request merged**